tests/ci/report_file_tests.js

```diff
@@ -0,0 +1,86 @@
+var fs = require('fs')
+var App = require('../../lib/ci')
+var TestReporter = require('../../lib/ci/test_reporters/tap_reporter')
+var Config = require('../../lib/config')
+var bd = require('bodydouble')
+var mock = bd.mock
+var stub = bd.stub
+var assert = require('chai').assert
+var expect = require('chai').expect
+var Process = require('did_it_work')
+var exec = require('child_process').exec
+var rimraf = require('rimraf')
+var path = require('path')
+var PassThrough = require('stream').PassThrough
+var ReportFile = require('../../lib/ci/report_file')
+var XUnitReporter = require('../../lib/ci/test_reporters/xunit_reporter')
+
+describe('report file output', function() {
+  var filename;
+  function getFileName() {
+    var timestamp = (+ new Date())
+    return 'test-results.' + timestamp + '.xml'
+  }
+  beforeEach(function() {
+    filename = getFileName()
+  })
+  afterEach(function(done) {
+    rimraf(filename, done)
```

Comment:
Sorry for my late response, but GitHub doesn't send a notification when only commits in a PR changed.
Could you check, whether the file exists? rimraf should do it, but doesn't seem to work.

Otherwise, you might need to use tempfiles for testing https://www.npmjs.com/package/tmp